site/docs/plugins/ratelimiter.md

```diff
@@ -46,51 +46,55 @@ There are two ways of using rateLimiter:
 
 ### Default Configuration
 
-The following example uses [express](https://github.com/expressjs/express) as the webserver and [webhooks](/guide/deployment-types) for our small bot.
-This snippet demonstrates the easiest way of using rateLimiter which is accepting the default behavior:
+This snippet demonstrates the easiest way of using rateLimiter, which is accepting the default behavior:
+
+<CodeGroup>
+  <CodeGroupItem title="Node.js" active>
 
 ```ts
-import express from "express";
-import { Bot } from "grammy";
 import { limit } from "@grammyjs/ratelimiter";
 
-const app = express();
-const bot = new Bot("YOUR BOT TOKEN HERE");
-
-app.use(express.json());
+// Limits message handling to a message per second for each user.
 bot.use(limit());
+```
+
+</CodeGroupItem>
+  <CodeGroupItem title="Deno">
 
-app.listen(3000, () => {
-  bot.api.setWebhook("YOUR DOMAIN HERRE", { drop_pending_updates: true });
-  console.log("The application is listening on port 3000!");
-});
+```ts
+import { limit } from "https://deno.land/x/grammy_ratelimiter/mod.ts";
+
+// Limits message handling to a message per second for each user.
+bot.use(limit());
 ```
 
+</CodeGroupItem>
+</CodeGroup>
+
 ### Manual Configuration
 
-As mentioned before, you can pass an `Options` object to the `limit()` method to alter rateLimiter's behaviors.
-In the following snippet, I have decided to use Redis as my storage option:
+As mentioned earlier, you can pass an `Options` object to the `limit()` method to alter the limiter's behavior.
+
+<CodeGroup>
+  <CodeGroupItem title="Node.js" active>
 
 ```ts
-import express from "express";
-import { Bot } from "grammy";
-import { limit } from "@grammyjs/ratelimiter";
 import Redis from "ioredis";
+import { limit } from "@grammyjs/ratelimiter";
 
-const app = express();
-const bot = new Bot("YOUR BOT TOKEN HERE");
-const redis = new Redis();
+const redis = new Redis(...);
 
-app.use(express.json());
 bot.use(
   limit({
+    // Allow only 3 messages to be handled every 2 seconds.
     timeFrame: 2000,
 
     limit: 3,
 
-    // "MEMORY_STORAGE" is the default mode. Therefore if you want to use Redis, do not pass storageClient at all.
+    // "MEMORY_STORE" is the default value. If you do not want to use Redis, do not pass storageClient at all.
     storageClient: redis,
 
+    // This is called when the limit is exceeded.
     onLimitExceeded: (ctx) => {
       ctx?.reply("Please refrain from sending too many requests!");
```

Comment:
May be necessary in further places.
```suggestion
      await ctx?.reply("Please refrain from sending too many requests!");
```

Reply:
Thanks, I commited modification to them!